src/ImageLoader/ImageLoader.js

```diff
@@ -0,0 +1,65 @@
+import SA from '../assets/AS.png';
+import SK from '../assets/KS.png';
+import SQ from '../assets/QS.png';
+import SJ from '../assets/JS.png';
+import S10 from '../assets/10S.png';
+import S9 from '../assets/9S.png';
+import S8 from '../assets/8S.png';
+import S7 from '../assets/7S.png';
+import S6 from '../assets/6S.png';
+import S5 from '../assets/5S.png';
+import S4 from '../assets/4S.png';
+import S3 from '../assets/3S.png';
+import S2 from '../assets/2S.png';
+
+import HA from '../assets/AH.png';
+import HK from '../assets/KH.png';
+import HQ from '../assets/QH.png';
+import HJ from '../assets/JH.png';
+import H10 from '../assets/10H.png';
+import H9 from '../assets/9H.png';
+import H8 from '../assets/8H.png';
+import H7 from '../assets/7H.png';
+import H6 from '../assets/6H.png';
+import H5 from '../assets/5H.png';
+import H4 from '../assets/4H.png';
+import H3 from '../assets/3H.png';
+import H2 from '../assets/2H.png';
+
+import DA from '../assets/AD.png';
+import DK from '../assets/KD.png';
+import DQ from '../assets/QD.png';
+import DJ from '../assets/JD.png';
+import D10 from '../assets/10D.png';
+import D9 from '../assets/9D.png';
+import D8 from '../assets/8D.png';
+import D7 from '../assets/7D.png';
+import D6 from '../assets/6D.png';
+import D5 from '../assets/5D.png';
+import D4 from '../assets/4D.png';
+import D3 from '../assets/3D.png';
+import D2 from '../assets/2D.png';
+
+import CA from '../assets/AC.png';
+import CK from '../assets/KC.png';
+import CQ from '../assets/QC.png';
+import CJ from '../assets/JC.png';
+import C10 from '../assets/10C.png';
+import C9 from '../assets/9C.png';
+import C8 from '../assets/8C.png';
+import C7 from '../assets/7C.png';
+import C6 from '../assets/6C.png';
+import C5 from '../assets/5C.png';
+import C4 from '../assets/4C.png';
+import C3 from '../assets/3C.png';
+import C2 from '../assets/2C.png';
+
+
+
+
+export const ImageLoader = [
+  SA, SK, SQ, SJ, S10, S9, S8, S7, S6, S5, S4, S3, S2,
+  HA, HK, HQ, HJ, H10, H9, H8, H7, H6, H5, H4, H3, H2,
+  DA, DK, DQ, DJ, D10, D9, D8, D7, D6, D5, D4, D3, D2,
+  CA, CK, CQ, CJ, C10, C9, C8, C7, C6, C5, C4, C3, C2,
+]
```

Comment:
I wish I could iterate over this and import, rather than having 52 imports above. Might look into this again.